### PR TITLE
feat: emit an event indicating that in-flight exit has finalized

### PR DESCRIPTION
**Pull request by pgebal**
Closes https://github.com/omgnetwork/plasma-contracts/issues/646

### CODE REVIEW

File: plasma_framework/contracts/src/exits/payment/controllers/PaymentProcessInFlightExit.sol [hunk 1]
```diff
@@ -123,6 +127,8 @@ library PaymentProcessInFlightExit {
             if (!success) {
                 emit InFlightBondReturnFailed(exit.bondOwner, exit.bondSize);
             }
+
+            emit InFlightExitFinalized(exitId);
```
Comment (InoMurko) [inline plasma_framework/contracts/src/exits/payment/controllers/PaymentProcessInFlightExit.sol:131]: whoa! how was this one missed. cool we finally cought it

**Comment by InoMurko** [comment]
@kevsul @souradeep-das should we do this too?

**Review by souradeep-das — APPROVED** [review]
Let's also generate solidoc before merge if we can?

File: plasma_framework/contracts/src/exits/payment/controllers/PaymentProcessInFlightExit.sol (hunk 1 above)
Comment (souradeep-das) [reply plasma_framework/contracts/src/exits/payment/controllers/PaymentProcessInFlightExit.sol:141]: Can we swap L141 and L142 🙈 if we add another commit

**Pull request merged**